aliases/available/git.aliases.bash

```diff
@@ -200,9 +201,5 @@ function gdv() {
 }
 
 function get_default_branch() {
-	if git branch | grep -q '^. main\s*$'; then
-		echo main
-	else
-		echo master
-	fi
+	git symbolic-ref refs/remotes/origin/HEAD | sed 's@^refs/remotes/origin/@@'
```

Comment:
Would it be possible to use a bash substitution instead of farming out to `sed`?

Reply:
Yes this is very much possible, good idea, changed it in the newest commit.
think I should squash it down later with the original commit if it looks good